docs/source/ko/_toctree.yml

```
@@ -164,8 +164,8 @@
     title: (번역중) The Transformer model family
   - local: in_translation
     title: (번역중) Summary of the tokenizers
-  - local: in_translation
-    title: (번역중) Attention mechanisms
+  - local: attention
+    title: (번역중) 어텐션 메카니즘
```

[표준국어대사전](https://stdict.korean.go.kr/search/searchView.do?word_no=115634&searchKeywordTo=3)에 등재된 단어로 수정 제안 드립니다
```suggestion
    title: (번역중) 어텐션 메커니즘
```